proto/buf/alpha/registry/v1alpha1/search_tag.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/repository_commit.proto";
+
+message SearchTagServiceSearchRequest {
+  // The owner of the repository.
+  string repository_owner = 1;
+  // The name of the repository.
+  string repository_name = 2;
+  // The search string.
+  string query = 3;
+  uint32 page_size = 4;
+  // The first page is returned if this is 0.
+  uint32 page_token = 5;
+}
+
+message SearchTagServiceSearchResponse {
+  repeated RepositoryCommit repository_commits = 1;
+  // There are no more pages if this is 0.
+  uint32 next_page_token = 2;
+}
+
+// SearchTagService is the search service that searches commits by tags
+service SearchTagService {
+  // Search searches in a repository
+  rpc Search(SearchTagServiceSearchRequest) returns (SearchTagServiceSearchResponse);
```

Comment:
We already have an existing `SearchService`, any reason to create another service specific for only tags?

https://github.com/bufbuild/buf/blob/main/proto/buf/alpha/registry/v1alpha1/search.proto#L103-L107

Could be easier and cleaner to add `SearchTags` method to the existing one, unless this was a FE requirement?